src/rpc/client/ts/src/viewport.ts

```diff
@@ -40,6 +40,7 @@ export function createViewport(
     request.setOffset(offset)
     request.setCapacity(capacity)
     request.setIsFloating(is_floating)
+    request.setEventInterest(ALL_EVENTS)
```

Comment:
@Shanedell, this will hopefully fix the issue you're seeing with data not getting populated in the viewport.  The code before wasn't subscribing to any events, now it's subscribed to all events.